notification/demo/README.md

```
@@ -0,0 +1,88 @@
+
+## 인프라 서비스
+
+### Kafka (포트 9092)
+- KRaft 모드로 실행 (Zookeeper 불필요)
+- 토픽: `notification.requests`, `notification.requests.DLQ`
+
+### Redis (포트 6379)
+- Idempotency 체크용
+
+### Kafka UI (포트 8080)
+- http://localhost:8080
```

markdownlint-cli2 (0.18.1)

Bare URL used

(MD034, no-bare-urls)